apps/ios/SimpleXChat/ChatUtils.swift

```diff
@@ -14,6 +14,45 @@ public protocol ChatLike {
     var chatStats: ChatStats { get }
 }
 
+extension ChatLike {
+    public func groupFeatureEnabled(_ feature: GroupFeature) -> Bool {
```

Comment:
can be reused between this and Chat? it's also ChatLike

Reply:
removed